pkgs/games/mudlet/darwin-AppKit.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 5b82401..2706722 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -528,6 +528,8 @@ if(APPLE)
+     target_link_libraries(mudlet cocoa-qt-glue "-framework AppKit")
```

Comment:
It seems there's already a conditional "-framework AppKit"?

Reply:
Yes, it should be visible in the patch#L7, but [the original CMakeLists.txt](https://github.com/Mudlet/Mudlet/blob/3bd76f24c81226338290e3295ff4833083b8bfd3/src/CMakeLists.txt#L566) only links with AppKit when the self-updater is enabled.
Some parts of AppKit, namely the _NSAccessibility* section is referenced a few places in the app, like https://github.com/Mudlet/Mudlet/blob/3bd76f24c81226338290e3295ff4833083b8bfd3/src/AnnouncerMac.mm.

This might be more of an upstream problem? I have not tried building it outside of nix, but it looks like an error in the original cmake files, since it will not build without the updater.

Reply:
Would you like to file a PR upstream dropping the conditions on `USE_UPDATER`? I can do it for you later this afternoon if you wish.

Reply:
Sure, I can do it, but thanks anyways!